vaadin-date-picker-flow-parent/vaadin-date-picker-testbench/src/main/java/com/vaadin/flow/component/datepicker/testbench/DatePickerElement.java

```diff
@@ -152,14 +154,18 @@ protected String getValue() {
     }
 
     /**
-     * Opens the overlay, sets the value to the inner input element as a string
-     * and closes the overlay. This simulates the user typing into the input and
-     * triggering an update of the value property.
+     * Emulates the user changing the value. This in practice means clearing
+     * {@code value} of the {@code input} element and then setting the new value
+     * by sending the respective characters + Enter via the Selenium
+     * {@code sendKeys} API.
+     *
+     * @param string
+     *            the value to set
      */
     public void setInputValue(String value) {
-        this.open();
-        setProperty("_inputValue", value);
-        this.close();
+        var input = $("input").first();
```

Comment:
I think using `var` is untypical in this codebase, let's write out the type instead.

Reply:
👍 Done.